cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -394,6 +394,13 @@ class CallSubqueryTest extends DocumentingTest {
           databaseStateBehavior = ClearState,
         ) { resultTable() }
       }
+      section("Restrictions") {
+        p("There are restrictions on a query that uses `CALL { ... } IN TRANSACTIONS`:")
+        p("""* A nested `CALL { ... } IN TRANSACTIONS` inside a `CALL { ... }` clause is not supported.
+            #* A `CALL { ... } IN TRANSACTIONS` in a `UNION` is not supported.
+            #* A `CALL { ... } IN TRANSACTIONS` after a write clause is not supported, unless that write clause is inside a `CALL { ... } IN TRANSACTIONS`.""".stripMargin('#'))
```

Comment:
```suggestion
            #* A `CALL { ... } IN TRANSACTIONS` after a write clause is not supported, unless that write clause is inside a `CALL { ... } IN TRANSACTIONS`.""".stripMargin('#')).
```

Reply:
There is already a period after that sentence, just before the tripe quotation mark.

Reply:
You are right! Please ignore this one then :)